common/short-hex-str/src/lib.rs

```diff
@@ -27,18 +31,18 @@ impl ShortHexStr {
     ///
     /// Returns `Err(InputTooShortError)` if the input byte slice length is less
```

Comment:
nit: could you update the comment to say "Returns Err(Error::InputTooShortError)"